src/api/print.api.js

```diff
@@ -16,6 +16,23 @@ const PRINTING_DEFAULT_POLL_INTERVAL = 2000 // interval between each polling of
 const PRINTING_DEFAULT_POLL_TIMEOUT = 600000 // ms (10 minutes)
 
 const SERVICE_PRINT_URL = `${API_SERVICES_BASE_URL}print3/print/default`
+
+class GeoAdminCustomizer extends BaseCustomizer {
+    constructor(layerNamesToExclude) {
```

Comment:
Could you add a little JSDoc to describe what type you expect with `layerNamesToExclude`?